sql/CMakeLists.txt

```diff
@@ -113,9 +113,9 @@ TARGET_LINK_LIBRARIES(sql ${MYSQLD_STATIC_PLUGIN_LIBS}
 # On Windows platform we compile in the clinet-side Windows Native Authentication
 # plugin which is used by the client connection code included in the server.
 #
-IF(WIN32)
-  ADD_DEFINITIONS(-DAUTHENTICATION_WIN)
-  TARGET_LINK_LIBRARIES(sql auth_win_client)
+IF(WIN32)
```

Comment:
Is this a Line endings problem?

Reply:
Yes, Windows endlines converted to Unix.